Dockerfile

```diff
@@ -2,14 +2,14 @@ FROM alpine:latest AS builder
 
 ARG PINOT_VERSION=0.7.1
 ARG JITPACK_REPO=hypertrace/incubator-pinot
-ARG JITPACK_TAG=hypertrace-0.7.1-7
+ARG JITPACK_TAG=hypertrace-0.7.1-8
```

Comment:
Curious how this tag fixes the problem. Could you link the source to this version or explain what is this?

Reply:
https://github.com/hypertrace/incubator-pinot/releases/tag/hypertrace-0.7.1-8